admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,542 @@
+* link:../admin_solutions/user_role_mgmt.html#control-user-roles[Apply roles to users and service accounts],
+enabling you to control what they are able to do in OpenShift projects and in
+the cluster.
+* link:../admin_solutions/user_role_mgmt.html#share-templates-cluster[Share templates with all projects in the cluster]
+* link:../admin_solutions/user_role_mgmt.html#create-cluster-admin[Create other cluster administrators]
+
+== Limiting and Monitoring Users and Projects
+
+[[setting-limits-users-projects]]
+=== Setting Limits for Users and Projects
+
+*How can I create limits for users and projects?*
+
+You can place limits within your OpenShift cluster using link:../admin_guide/quota.html[ResourceQuotas]
+and
+link:../admin_guide/limits.html[LimitRanges],
+These quotas and limits allow you to control pod and container limits, object
+counts, and compute resources. Currently, these limits and quotas only apply to
+projects and not to users. However, you can make a quota-like
+link:../admin_solutions/user_role_mgmt.html#limit-number-projects[limit on how many project requests a user can make].
+
+*Use Case: I want to create a quota in my "awesomeproject", so that there is a maximum of only 10 pods that can be created in the project.*
+
+. Create a *_resource-quota.yaml_* file with the following contents:
++
+----
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-resources
+spec:
+  hard:
+    pods: "10"
+----
++
+. Create the quota using the file you just wrote to apply it to the "awesomeproject":
++
+----
+$ oc create -f resource-quota.yaml -n awesomeproject
+----
++
+. After the quota has been in effect for a little while, you can view the usage statistics for the hard limit set on pods. If required, list the quotas defined in the project to see the names of all defined quotas:
++
+====
+----
+$ oc get quota -n awesomeproject
+NAME                AGE
+resource-quota      39m
+----
+====
++
+. Describe the resource quota for which you want statistics:
++
+====
+----
+$ oc describe quota resource-quota -n awesomeproject
+Name:			resource-quota
+Namespace:		awesomeproject
+Resource		Used	Hard
+--------		----	----
+pods     		3	10
+----
+====
++
+. Optionally, you can
+link:../admin_guide/quota.html#configuring-quota-sync-period[configure the quota synchronization period], which controls how long to wait before restoring quota usage after resources are deleted.
+. If you want to remove an active quota to no longer enforce the limits of a project:
++
+----
+$ oc delete quota <quota_name>
+----
+
+*Now let's walk through that process and examine all the available options:*
+
+. To create project limits or quotas, you must first write a .yaml file that describes the resource that you want to restrict:
+
+* This *_LimitRange.yaml_* example explains all the
+link:../dev_guide/limits.html#container-limits[container limits] and
+link:../dev_guide/limits.html#pod-limits[pod limits] that you can place within
+your project:
```

Comment:
Well actually, they should probably go to the admin_guide version (the content is shared between the two because it's relevant to both users and admins):

../admin_guide/limits.html#container-limits
../admin_guide/limits.html#pod-limits